Description: Patch the sage startup script
 Set required environment variables and 
 remove a check for being in a sage source directory.
Author: Tobias Hansen <thansen@debian.org>
Forwarded: not-needed

--- a/sage/src/bin/sage
+++ b/sage/src/bin/sage
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+if [ "$0" = "/usr/bin/sage" ]; then
+    export SAGE_LOCAL="/usr"
+    export SAGE_SCRIPTS_DIR="/usr/share/sagemath/bin"
+    export SAGE_PYTHON3=yes
+    export PATH="$PATH:$SAGE_SCRIPTS_DIR"
+fi
+
 # Display the current version of Sage
 # usage: sage_version [-v]
 #   -v    display the full version banner including release date
@@ -217,26 +224,6 @@
 
 # Prepare for running Sage, either interactively or non-interactively.
 sage_setup() {
-    # Check that we're not in a source tarball which hasn't been built yet (#13561).
-    if [ ! -z "$SAGE_LOCAL" ] && [ ! -x "$SAGE_LOCAL/bin/sage" ]; then
-        echo >&2 '************************************************************************'
-        echo >&2 'It seems that you are attempting to run Sage from an unpacked source'
-        echo >&2 'tarball, but you have not compiled it yet (or maybe the build has not'
-        echo >&2 'finished). You should run `make` in the Sage root directory first.'
-        echo >&2 'If you did not intend to build Sage from source, you should download'
-        echo >&2 'a binary tarball instead. Read README.txt for more information.'
-        echo >&2 '************************************************************************'
-        exit 1
-    fi
-
-    maybe_sage_location
-
-    if [ ! -d "$IPYTHONDIR" ]; then
-        # make sure that $DOT_SAGE exists so that ipython will happily
-        # create its config directories there.  If DOT_SAGE doesn't
-        # exist, ipython complains.
-        mkdir -p -m 700 "$DOT_SAGE"
-    fi
     sage-cleaner &>/dev/null &
 }
 
